ingresspipe/synapse/store.py

```diff
@@ -444,23 +442,4 @@ def associateMetadataWithFiles(self, metadataManifestPath: str, datasetId: str)
 
         manifestSynapseFileId = self.syn.store(manifestSynapseFile).id
 
-        return manifestSynapseFileId
-
-# if __name__ == "__main__":
-#     # create an instance of synapseclient.Synapse() and login
-#     syn = synapseclient.Synapse()
-
-#     try:
-#         syn.login("sujaypatil", "Sujay123@#@")
```

Comment:
Might want to change your Synapse password...

Reply:
Will do. Thanks for pointing it out! The new file(s) that will be merged with this PR once @xdoan has reviewed them does not have the above lines.